build.sbt

```diff
@@ -28,7 +28,8 @@ libraryDependencies ++= Seq(
   "org.foundweekends" %% "pamflet-library" % "0.7.1",
   "org.yaml"        % "snakeyaml"        % "1.13",
   "com.typesafe"    % "config"           % "1.2.1", // Last version to support Java 1.6
-  "org.asciidoctor" % "asciidoctorj"     % "1.5.4"
+  "org.asciidoctor" % "asciidoctorj"     % "1.5.4",
+  "org.asciidoctor" % "asciidoctorj-diagram" % "1.5.4.1"
```

Comment:
Why is this using a different version?

Reply:
The different version wasn't intentional.

Should I downgrade the asciidoctorj-diagram dependency to 1.5.4, or upgrade the asciidoctorj dependecy to 1.5.4.1?

1.5.4.1 is a bugfix release https://github.com/asciidoctor/asciidoctorj/releases/tag/v1.5.4.1